--- dbus_digitalinputs.py	2019-03-21 19:46:47.639634076 +0100
+++ dbus_digitalinputs.py.new	2019-03-21 19:46:15.419532076 +0100
@@ -35,7 +35,9 @@
     'Burglar alarm',
     'Smoke alarm',
     'Fire alarm',
-    'CO2 alarm'
+    'CO2 alarm',
+    'Generator',
+    'Inverter control'
 ]
 
 # Translations. The text will be used only for GetText, it will be translated
@@ -356,6 +358,10 @@
     type_id = 9
     translation = 5 # running, stopped
 
+class InverterControl(PinAlarm):
+    _product_name = "Inverter control"
+    type_id = 10 
+    translation = 1 #on, off
 
 def dbusconnection():
     return SessionBus() if 'DBUS_SESSION_BUS_ADDRESS' in os.environ else SystemBus()
