--- MbItemDigitalInput.qml	2023-07-18 09:56:03.755904477 +0200
+++ MbItemDigitalInput.qml.new	2023-07-18 09:58:00.506023081 +0200
@@ -15,7 +15,8 @@
 		MbOption { description: qsTr("Fire alarm"); value: 7 },
 		MbOption { description: qsTr("CO2 alarm"); value: 8 },
 		MbOption { description: qsTr("Generator"); value: 9 },
-		MbOption { description: qsTr("Touch input control"); value: 11 }
+		MbOption { description: qsTr("Touch input control"); value: 11 },
+		MbOption { description: qsTr("Inverter control"); value: 12 }
 	]
 	onValueChanged: {
 		if (valid) {
