--- MbItemDigitalInput.qml	2023-07-17 12:18:32.386891523 +0200
+++ MbItemDigitalInput.qml.new	2023-07-17 12:19:33.806295994 +0200
@@ -15,6 +15,7 @@
 		MbOption { description: qsTr("Fire alarm"); value: 7 },
 		MbOption { description: qsTr("CO2 alarm"); value: 8 },
 		MbOption { description: qsTr("Generator"); value: 9 },
+		MbOption { description: qsTr("Inverter control"); value: 10 },
 		MbOption { description: qsTr("Touch input control"); value: 11 }
 	]
 	onValueChanged: {
